docker/debug/pod.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: gp-v2
+
+spec:
+  containers:
+    - name: solver
+      image: gnosispm/gp-v2-services
+      args: ["solver"]
+      env:
+        - name: PRIVATE_KEY
+          value: "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
+        - name: AMM_SOURCES
+          value: Uniswap
```

Comment:
Why that restriction? Because sushi swap is not yet deployed?

Reply:
We don't deploy SushiSwap on Ganache. (with `cargo run --bin deploy`).